P/Pango/build_tarballs.jl

```diff
@@ -47,12 +47,13 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    "FriBidi_jll",
-    "FreeType2_jll",
-    "Glib_jll",
-    "Fontconfig_jll",
-    "HarfBuzz_jll",
-    "Cairo_jll",
+    Dependency("FriBidi_jll"),
+    Dependency("FreeType2_jll"),
+    Dependency("Glib_jll"),
+    Dependency("Fontconfig_jll"),
+    Dependency("HarfBuzz_jll"),
+    Dependency("Cairo_jll"),
+    BuildDependency("Xorg_xorgproto_jll")
```

Comment:
@giordano this happened twice today, might it be a common case where we recursively need build dependency or is it only Cairo weirdness?

Reply:
Cairo [does need `Xorg_xorgproto_jll`](https://github.com/JuliaPackaging/Yggdrasil/commit/4189d37aa04c7821332b71c66d86112fe8fcbfbe), probably building libpangocairo needed it as well.

Having to explicitly mention these build-time dependencies is to be expected, as they aren't pulled automatically by other packages -- and if they're indeed needed it's good to list them explicitly anyway, instead of relying on other packages pulling them.